seismicpro/src/plot_utils.py

```diff
@@ -87,42 +137,44 @@ def seismic_plot(arrs, wiggle=False, xlim=None, ylim=None, std=1, # pylint: disa
                 for ix, k in enumerate(offsets):
                     x = k + std * arr[k, slice(*ylim_curr)] / np.std(arr)
                     col = line_color[ix]
-                    ax[0, i].plot(x, y, '{}-'.format(col))
-                    ax[0, i].fill_betweenx(y, k, x, where=(x > k), color=col)
+                    ax.plot(x, y, '{}-'.format(col))
+                    ax.fill_betweenx(y, k, x, where=(x > k), color=col)
 
             else:
-                ax[0, i].imshow(arr.T, **kwargs)
+                setup_imshow(ax, arr.T, **kwargs)
+        
+            if attribute is not None:
+                scatter_on_top(ax, attribute.flatten()[i])
 
         elif arr.ndim == 1:
-            ax[0, i].plot(arr, **kwargs)
+            ax.plot(arr, **kwargs)
         else:
             raise ValueError('Invalid ndim to plot data.')
 
-        if names is not None:
-            ax[0, i].set_title(names[i])
+        # if names is not None:
+        #     ax.set_title(names[i])
 
         if arr.ndim == 2:
-            ax[0, i].set_ylim([ylim_curr[1], ylim_curr[0]])
+            ax.set_ylim([ylim_curr[1], ylim_curr[0]])
             if (not wiggle) or (pts is not None):
-                ax[0, i].set_xlim(xlim_curr)
+                ax.set_xlim(xlim_curr)
 
         if arr.ndim == 1:
             plt.xlim(xlim_curr)
 
         if pts is not None:
-            ax[0, i].scatter(*pts, s=s, c=scatter_color)
-
-        ax[0, i].set_aspect('auto')
+            ax.scatter(*pts, s=s, c=scatter_color)
 
     if title is not None:
         fig.suptitle(title)
+
     if save_to is not None:
-        plt.savefig(save_to, dpi=dpi, transparent=True)
+        plt.savefig(save_to, dpi=dpi)
```

Comment:
condiser to change all savefigs to
```suggestion
        plt.savefig(save_to, dpi=dpi, bbox_inches='tight', pad_inches=0.1)
```